Packs/Graylog/Integrations/Graylog/README.md

```diff
@@ -0,0 +1,1205 @@
+***
+Shows the processing status of the cluster
+
+
+#### Base Command
+
+`graylog-cluster-processing-status`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| Graylog.ClusterProcessingStatus | String | Processing status of the cluster | 
+
+
+#### Command Example
+```!graylog-cluster-processing-status```
+
+#### Context Example
+```json
+{
+    "Graylog": {
+        "ClusterProcessingStatus": {
+            "95ba5102-13c9-4520-ac75-c8736f206953": {
+                "receive_times": {
+                    "ingest": "2020-10-08T10:08:29.353Z",
+                    "post_indexing": "2020-10-08T10:08:29.353Z",
+                    "post_processing": "2020-10-08T10:08:29.353Z"
+                }
+            }
+        }
+    }
+}
+```
+
+#### Human Readable Output
+
+>### Results
+>|95ba5102-13c9-4520-ac75-c8736f206953|
+>|---|
+>| receive_times: {"ingest": "2020-10-08T10:08:29.353Z", "post_processing": "2020-10-08T10:08:29.353Z", "post_indexing": "2020-10-08T10:08:29.353Z"} |
+
+
+### graylog-indexer-cluster-health
+***
+Get health of the indexer
+
+
+#### Base Command
+
+`graylog-indexer-cluster-health`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| Graylog.IndexerHealth | String | Health of Indexer | 
+
+
+#### Command Example
+```!graylog-indexer-cluster-health```
+
+#### Context Example
+```json
+{
+    "Graylog": {
+        "IndexerHealth": {
+            "shards": {
+                "active": 20,
+                "initializing": 0,
+                "relocating": 0,
+                "unassigned": 0
+            },
+            "status": "green"
+        }
+    }
+}
+```
+
+#### Human Readable Output
+
+>### Results
+>|shards|status|
+>|---|---|
+>| active: 20<br/>initializing: 0<br/>relocating: 0<br/>unassigned: 0 | green |
+
+
+### graylog-search
+***
+Search for messages in a relative timerange, specified as seconds from now. Example: 300 means search from 5 minutes ago to now.
+
+
+#### Base Command
+
+`graylog-search`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| query | Query (Lucene syntax) | Required | 
+| range | Relative timeframe to search in. Default 300s | Optional | 
+| limit | Maximum number of messages to return. Default 20 | Optional | 
+| offset | offset (integer) | Optional | 
+| filter | filter | Optional | 
+| fields | Comma separated list of fields to return | Optional | 
+| sort | Sorting (field:asc / field:desc) | Optional | 
+| decorate | Run decorators on search result (default True) | Optional | 
+
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| Graylog.Search | String | Search results | 
+
+
+#### Command Example
+```!graylog-search query=<query here>```
```

Comment:
One of the errors raised in `ci/circleci: Run Validations` is that `<query here>` is perceived as a part of an HTML code that isn't closed. In order to remedy the error please replace `<query here>` with `\<query here\>` everywhere it appears.